libs/src/sdk/services/Storage/types.ts

```diff
@@ -1,8 +1,7 @@
 import type { CrossPlatformFile as File } from '../../types/File'
+import type { StorageNodeSelectorConfig } from './StorageNodeSelector'
 
-export type StorageServiceConfig = {
-  contentNodeEndpoint: string
-}
+export type StorageServiceConfig = StorageNodeSelectorConfig
```

Comment:
Should we maybe nest the selector config, in case we want to add config specific to StorageService in the future?

Reply:
updated